fetch-quotes.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable indent */
+import fetch from 'node-fetch';
+
+const fetchQuotes = async () => {
+    
+    const apiRequest = await fetch('https://futuramaapi.herokuapp.com/api/quotes');
+
+    const body = await apiRequest.json();
+
+return body.results;
```

Comment:
As mentioned in the test file, you'll find that body.results is undefined. I'd recommend updating your test according to the comments I've left there and then using the more robust test to guide reworking this function to get the result you want.